### ISSUE TITLE
Keyboard mode switch is not working

### ISSUE THREAD

**Issue by BGPierce** [VCCE/VCC#10]
I had a VCC 201 user report that the keyboard modes "Basic" and "OS9" are not functioning properly. It seems that no matter what the setting, the keyboard mode stays in OS9 mode.
I tried this on my system (Win Vista Home Premium 64bit) and found the same problem.
This seems to fall along with a previously reported bug (by me) that the "Overclock Disk Drives" is not being persistant from one run to the next.

I'm wondering if there any other buttons, pull downs or check boxes not functionng properly.
I will start running some checks and see what I find.

Bill P.

**Comment by BGPierce** [comment]
I finally got to compile the latest merge in the master branch and I find this bug still persists.
It 'may' be my Vista machine as I have found several things working strangely in Vista, but I'm not sure yet. I will test on windows 8 and see what I find there. I also have someone who can test on an old XP machine. I will send them a copy.
The easiest test for this bug is:
Basic Keyboard selection: "<SHIFT> 2" should produce a "&" on the screen
OS9 Keyboard selection: "<SHIFT> 2" should produce a "@" on the screen.
Currently on my Vista system, both selections (Basic & OS9) produce "@" on the screen.
This pulldown selection is in the "configuration/keyboard" menu.
B.P.
